XMonad/Actions/Prefix.hs

```diff
@@ -136,6 +143,13 @@ usePrefixArgument prefix conf = conf {
           let binding = prefix conf'
           in M.singleton binding (handlePrefixArg [binding])
 
+-- | Like 'usePrefixArgument', but allow the key to be entered in Emacs
+-- style syntax.  Defaults to @C-u@ when given an invalid key.
+easyPrefixArgument :: LayoutClass l Window => String -> XConfig l -> XConfig l
+easyPrefixArgument prefix conf = case readKeySequence conf prefix of
+  Just [key] -> usePrefixArgument (const key) conf
+  _          -> useDefaultPrefixArgument conf
+
```

Comment:
I actually think the better change would be to just use this as the implementation of `usePrefixArgument`—using the EZConfig parser seems like a straight upgrade to having to manually extract the mod key from an `XConfig` argument

Reply:
On second thought, I really do think this is the right decision and I've made the appropriate changes